src/Avalonia.Base/Input/GestureRecognizers/ScrollGestureRecognizer.cs

```diff
@@ -58,6 +61,30 @@ public class ScrollGestureRecognizer : GestureRecognizer
                 o => o.ScrollStartDistance, (o, v) => o.ScrollStartDistance = v,
                 unsetValue: s_defaultScrollStartDistance);
 
+        /// <summary>
+        /// Defines the <see cref="ScrollStartDistance"/> property.
+        /// </summary>
+        public static readonly DirectProperty<ScrollGestureRecognizer, Vector?> OffsetProperty =
```

Comment:
Do we need those properties to be settable outside of the ScrollGestureRecognizer? They are not read-only rn.